### DISCUSSION TITLE
Enable sponsors on this repository

### DISCUSSION THREAD

**Discussion by spacentropy** [mysql-net/MySqlConnector#1027] [Ideas]
Hi! Can you please enable sponsors on this repository?

**Comment by bgrainger** [comment]
Are you interested in a one-time, or monthly recurring sponsorship?

What approximate sponsorship amount are you considering? $5, $10, $50, $100, $500, $1000, ?

What kind of "reward" are you interested in? No reward, just saying thanks; personal thank you in README; corporate logo on mysqlconnector.net; priority bug/tech support; dedicated development work on a specific task; etc.
